src/shared/apis/records/getStatistics.ts

```diff
@@ -0,0 +1,16 @@
+import { createQuery } from 'react-query-kit';
+
+import { Statistics } from '@/shared/types/record/statistics';
+import { request } from '@/shared/utils/request';
+
+const getStatistics = () => {
+  return request<Statistics>({
+    method: 'get',
+    url: 'records/me/statistics',
+  });
+};
+
+export const useGetStatistics = createQuery({
+  primaryKey: '/statistics',
+  queryFn: () => getStatistics(),
```

Comment:
취향차이이지만 이렇게만 적어줘도 될 것 같습니다!
```suggestion
  queryFn: getStatistics,
```